README.md

```diff
@@ -12,15 +12,14 @@
 - Twitter: [@jsivahera](https://twitter.com/jsivahera)
 - LinkedIn: [jsivahera](https://www.linkedin.com/in/jsivahera/)
 
-👤 **Author 2**
-
-- GitHub: [name](https://github.com)
-- Twitter: [name](https://twitter.com)
-- LinkedIn: [name](https://www.linkedin.com/in)
+👤 **Samiullah Bahadur**
+- GitHub: [@ Bahadur](https://github.com/samiullahbahadur)
+- Twitter: [@Bahadur](https://twitter.com/Samiull88496331)
+- LinkedIn: [Bahadur](https://www.linkedin.com/in/jsivahera/)
 
 👤 **Author 3**
 
-- GitHub: [name](https://github.com)
-- Twitter: [name](https://twitter.com)
-- LinkedIn: [name](https://www.linkedin.com/in)
+- GitHub: [@bushra](https://github.com/bushmusi)
+- Twitter: [@bushra](https://twitter.com/bushera_mestofa)
+- LinkedIn: [@bushra](https://www.linkedin.com/in/bushra-mustofa)
```

Comment:
- [ ] please make sure to use [the template](https://github.com/microverseinc/readme-template) for the readme file provided by Microverse, including the links (for example to the MIT license). keep in mind that all sections of the template are mandatory except for the "Live Demo", "Run Tests" and "Deployment" sections, these are only mandatory if the specific project you're working on requires it.

Reply:
Hi @mikethreels 

Thanks for your comment, noted and we will fix them